PrivBox/README.md

```diff
@@ -0,0 +1,30 @@
+# PrivBox--Privacy Analysis Tools
+English | [简体中文](./README_cn.md)
+
+
+PrivBox is a Python library for testing AI security and privacy, which is based on PaddlePaddle. PrivBox provides multiple AI attack implementations of recent researches, including model inversion, membership inference, property inference and model extraction, aiming to help developer to evaluate security and privacy of AI models.
```

Comment:
testing AI security and privacy -> testing AI model privacy leaking risk
multiple AI attack -> multiple AI privacy attacks
security and privacy of AI models -> find out privacy issues of AI models.